app/models/user.rb

```diff
@@ -37,4 +38,18 @@ def self.from_omniauth(auth)
   #     end
   #   end
   # end
+
+  def self.connect_to_person(auth, id)
+    email = auth.info.email.gsub("@", "(at)")
```

Comment:
Should we really replace the @ with (at)? I think long term, we should replace the (at) from the scraper with @, right?

Reply:
yeah right, but than lets change it after demonstration tomorrow in the rest of the sprint since some parts will need to be touched

Reply:
Ok (@HerzogVonWiesel -> don't create a person for yourself before demonstrating the HPI Open connect linking, or keep in mind that it will only link if you enter the `@` in your email as `(at)`)